### PR TITLE
Update funfacts.txt

### PR DESCRIPTION
**Pull request by Zetvue**

### CODE REVIEW

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
...
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+KTibow holds the record for the fastest promotion to admin.
+Play The Binding of Isaac instead of Hypixel Skyblock
+Play Terraria instead of Hypixel Skyblock
+Play Minecraft Survival instead of Hypixel Skyblock
+Play literally anything else instead of Hypixel Skyblock
+Just 30 minutes of daily complaining can physically damage your brain.\nListening to someone constantly complain also negatively affects brain function.
+The # symbol isn’t officially called a hashtag or a pound.\nIt’s technically called an "octothorpe." The “octo-” means “eight” to refer to its points.
+Try to do something creative! It could be music, art, coding...\nYou won't know how much you like it until you try it, so give it a shot :)
+Minecraft 1.8.9 is almost 7 years old.\nWhy are you playing severely outdated versions again?
```
Comment (KTibow) [inline docs/assets/funfacts.txt:33]: maybe just replace this with https://howoldisminecraft189.today/

File: docs/assets/funfacts.txt [hunk 2]
```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
 You'll never believe this, but our Discord's support channels\nare in a category called SUPPORT IS HERE! Use them as necessary.
 SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
+In Canada, Santa's postal code is H0H 0H0.
+Zetvue makes music! You can too!
+Guvf vf abg na vzcbegnag fragrapr.
+ROT13
+Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
```
Comment (KTibow) [inline docs/assets/funfacts.txt:23]: you know these aren't displayed in order or anything

File: docs/assets/funfacts.txt [hunk 3]
```diff
@@ -16,6 +16,21 @@ SkyClient was once exclusive to Windows users.\nFortunately, now we have a Java-
...
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```
Comment (KTibow) [inline docs/assets/funfacts.txt:24]: given this is literal screaming you should make the first one as long as the second one

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:23]: i know

File: docs/assets/funfacts.txt (hunk 3 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:24]: the AAAAAAAAAA might go offscreen so thats why one is shorter than the other

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:33]: no

File: docs/assets/funfacts.txt (hunk 3 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:24]: as in it's good if it's offscreen bc chaos

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:23]: well it's going to be really confusing

File: docs/assets/funfacts.txt (hunk 3 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:24]:
```suggestion
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:33]: the way this is phrased sounds weird
having a link to that site is more neutral but also could be interpreted in more ways

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:23]: that's the point
anyone that knows ROT13 will know lol

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (Zetvue) [reply docs/assets/funfacts.txt:33]: eh nah

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:23]:
```suggestion
Guvf vf abg na vzcbegnag fragrapr.
Vs lbh qrpvcure guvf, lbh fubhyq yvfgra gb Mrgihr'f zhfvp.
```
also you only get one ad

**Pull request merged**